### PR TITLE
Added Haitian Creole translation

### PR DESCRIPTION
**Pull request by farahbrunache**
Addresses issue: https://github.com/MetaMask/metamask-extension/issues/5141

### CODE REVIEW

**Comment by mapmeld** [comment]
Hi - I've been looking for someone who can give a thumbs-up on the translation, but haven't gotten detailed feedback as of yet.

Via Telegram:
> Overall it's good, a few grammatical errors

[Via Twitter](https://twitter.com/uncle_ernst/status/1037084402965245952)
> A few suggestions. On line 12 I’d flip the 2 words around. Instead of “kont detay” I’d put “detay kont”. On line 18 I’d replace “bezwen” with “dwe”. On lines 24, 27, 30, and 33 I’d replace “add” with “ajoute”. I didn’t go through the entire thing

Do those changes make sense?

Also "$ 1" and "$ 2" should be written "$1" without a space - these are not dollar amounts but placeholders which will be replaced by text

**Comment by farahbrunache** [comment]
@mapmeld, I am in Haiti now, arrived a few days ago. I am working out of a co-working space and I will have someone double check all the translations.

I will also make the updates from the Tweet, the suggestions make sense!

**Comment by mapmeld** [comment]
@farahbrunache mesi anpil!  It's been a few years since I was in Haiti, hope all is well.  Looking forward to seeing this added into MetaMask.

File: app/_locales/ht/messages.json [hunk 1]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Non Kont"
+  },
+  "accountSelectionRequired": {
+    "message": "Ou bezwen chwazi yon kont!"
+  },
+  "address": {
+    "message": "Adrès"
+  },
+  "addCustomToken": {
+    "message": "Add koutim token"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: "ajoute koutim token"

File: app/_locales/ht/messages.json [hunk 2]
```diff
@@ -0,0 +1,1183 @@
...
+  },
+  "addToken": {
+    "message": "Add Token"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: "Ajoute Token"

File: app/_locales/ht/messages.json [hunk 3]
```diff
@@ -0,0 +1,1183 @@
...
+  },
+  "addTokens": {
+    "message": "Add Tokens"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: "Ajoute Tokens"

File: app/_locales/ht/messages.json [hunk 4]
```diff
@@ -0,0 +1,1183 @@
...
+  },
+  "addAcquiredTokens": {
+    "message": "Add tokens yo ou te achte lè l sèvi avèk MetaMask"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: Replace "add" for "ajoute"

File: app/_locales/ht/messages.json [hunk 5]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Kantite lajan + Gaz"
+  },
+  "appDescription": {
+    "message": "Ethereum Ekstansyon Navigatè",
+    "description": "The description of the application"
+  },
+  "appName": {
+    "message": "MetaMask",
+    "description": "The name of the application"
+  },
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: "Non aplikasyon an"

File: app/_locales/ht/messages.json [hunk 6]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Itilize Blockies Identicon"
+  },
+  "borrowDharma": {
+    "message": "Prete Avèk Dharma (Beta)"
+  },
+  "browserNotSupported": {
+  "message": "Navigatè ou a pa sipòte..."
+  },
+  "builtInCalifornia": {
+    "message": "MetaMask se fèt ak bati nan California."
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: remove "se"

File: app/_locales/ht/messages.json [hunk 7]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Kopye!"
+  },
+  "copiedSafe": {
+    "message": "Mwen te kopye li yon kote ki san danje"
+  },
+  "copy": {
+    "message": "Kopye"
+  },
+  "copyContractAddress": {
+    "message": "Kopi Kontra Adrès"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: "Kopi Adrès Kontra"

File: app/_locales/ht/messages.json [hunk 8]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Kopi adrès clipboard"
+  },
+  "copyToClipboard": {
+    "message": "Kopi clipboard"
+  },
+  "copyButton": {
+    "message": " Kopi "
+  },
+  "copyPrivateKey": {
+    "message": "Sa a se kle prive ou (klike sou yo kopye)"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: remove "sou yo" replace with "pou"

File: app/_locales/ht/messages.json [hunk 9]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Antre nan modpas ou a konfime"
+  },
+  "enterPasswordContinue": {
+    "message": "Mete modpas pou kontinye"
+  },
+  "parameters": {
+    "message": "Paramèt"
+  },
+  "passwordNotLongEnough": {
+    "message": "Modpas pa lontan ase"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: "Modpas la pa ase longè"

File: app/_locales/ht/messages.json [hunk 10]
```diff
@@ -0,0 +1,1183 @@
...
+  },
+  "passwordsDontMatch": {
+    "message": "Modpas Pa Koresponn ak"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: "Modpas yo pa matche"

File: app/_locales/ht/messages.json [hunk 11]
```diff
@@ -0,0 +1,1183 @@
...
+  },
+  "etherscanView": {
+    "message": "Gade kont sou Etherscan"
+  },
+  "exchangeRate": {
+    "message": "Echanj to"
+  },
+  "exportPrivateKey": {
+    "message": "Export Prive Key"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: "Export kle prive"

File: app/_locales/ht/messages.json [hunk 12]
```diff
@@ -0,0 +1,1183 @@
...
+  },
+  "exportPrivateKeyWarning": {
+    "message": "Ekspòte kle prive nan pwòp risk ou."
+  },
+  "failed": {
+    "message": "Failed"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: "echwe"

File: app/_locales/ht/messages.json [hunk 13]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Rezo"
+  },
+  "nevermind": {
+    "message": "Bliye li"
+  },
+  "newAccount": {
+    "message": "Nouvo Kont"
+  },
+  "newAccountNumberName": {
+    "message": "Kont $1",
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: NonNouvoNimewoKont

File: app/_locales/ht/messages.json [hunk 14]
```diff
@@ -0,0 +1,1183 @@
...
+  },
+  "passwordCorrect": {
+    "message": "Tanpri asire ke modpas ou kòrèk."
+  },
+  "passwordMismatch": {
+    "message": "modpas pa matche",
+    "description": "in password creation process, the two new password fields did not match"
+  },
+  "passwordShort": {
+    "message": "modpas pa lontan ase",
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: replace "lontan" with "llongè"

File: app/_locales/ht/messages.json [hunk 15]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Popilè Tokens"
+  },
+  "prev": {
+    "message": "Prev"
+  },
+  "privacyMsg": {
+    "message": "Règleman sou enfòmasyon prive"
+  },
+  "privateKey": {
+    "message": "Prive kle",
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: kle Prive

File: app/_locales/ht/messages.json [hunk 16]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Mete kòd QR la devan kamera ou"
+  },
+  "scanQrCode": {
+    "message": "Enspeksyon QR Kòd"
+  },
+  "shapeshiftBuy": {
+    "message": "Achte avèk Shapeshift"
+  },
+  "showPrivateKeys": {
+    "message": "Montre Keys Prive"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: replace "keys" with "kle"

File: app/_locales/ht/messages.json [hunk 17]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Pran twò lontan?"
+  },
+  "terms": {
+    "message": "Tèm itilizasyon"
+  },
+  "testFaucet": {
+    "message": "Tès Robo"
+  },
+  "to": {
+    "message": "Yo"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: pou

File: app/_locales/ht/messages.json [hunk 18]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Pa apwouve"
+  },
+  "unavailable": {
+    "message": "Pa disponib"
+  },
+  "unknown": {
+    "message": "Enkoni"
+  },
+  "unknownFunction": {
+    "message": "Enkoni fonksyon"
```
Comment (QueenT03) [inline app/_locales/ht/messages.json]: fonksyon Enkoni

**Comment by farahbrunache** [comment]
@QueenT03 and @mapmeld. I am at line 817 the individual whom is reviewing my work will be available tomorrow. Please give me another day and I will have the revisions done.

@QueenT03, thank you for the suggestions the individual who is helping me caught them as well. But she is non-technical and doesn't have Git. I'll push all the changes tomorrow. Do you mind taking a look at the pushed changes tomorrow and onwards instead? This way you both won't be correcting the same errors at the same time ... :).

**Comment by farahbrunache** [comment]
I will submit a new pull request with the language translation. Likely tomorrow.

File: app/_locales/ht/messages.json [hunk 19]
```diff
@@ -0,0 +1,1183 @@
...
+    "message": "Avètisman: Pa janm divilge kle sa a. Nenpòt moun ki gen kle prive ou ka pran vòlè nenpòt byen ki te fèt nan kont ou."
+  },
+  "privateNetwork": {
+    "message": "Rezo Prive"
+  },
+  "qrCode": {
+    "message": "Montre QR Kòd"
+  },
+  "readdToken": {
+    "message": "Ou ka ajoute sa a siy tounen nan tan kap vini an pa pral ale nan "Add token" nan meni opsyon kont ou an."
```
[the remark this answers is not in the document]
Comment (farahbrunache) [reply app/_locales/ht/messages.json]: @mapmeld, I missed the memo that I could push to an existing pull request of mine. Sorry for the confusion. I had created a new one a few days ago. Please see here: https://github.com/MetaMask/metamask-extension/pull/5271.

Bullet one is already addressed.

I will check pull #5271 for what is addressed in the second bullet.

Third bullet is correct, singular and plural form are the same.

[the remark this answers is not in the document]
Comment (farahbrunache) [reply app/_locales/ht/messages.json]: Pull request #5271 has fixed the double quote issue.

**Pull request closed without merging**